.github/workflows/validation-check.yaml

```diff
@@ -47,4 +66,4 @@ jobs:
               }
             } else {
               core.setFailed("No validation comments by github-actions[bot] found.");
-            }
+            }
```

Comment:
Nit: do we want a newline at the end of this file?

Reply:
I don't know if there is a convention for this. I know it does not break, we can discuss it.

Reply:
On previous teams this has been something that others mentioned on my PRs, which is what caused me to mention it. But I realized I didn't understand why people want this, so I did a little searching and found this: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

TLDR it is a unix thing, and has to do with certain utilities not working as expected in some cases if the newline is not present.

I don't think it's likely that we will run into problems because of this, but since we're here, we might as well fix it.

Reply:
Ok, I was updating the other propagation PRs before I read this message. 
I will make this change on the other PRs as well.